js-plugins/mcp/src/types.ts

```
@@ -0,0 +1,71 @@
+  printUrl?: boolean
+
+  /**
+   * The MCP server info. Ingored when `mcpServer` is provided
+   */
+  mcpServerInfo?: McpServerInfo
+
+  /**
+   * Setup the MCP server, this is called when the MCP server is created
+   * You may also return a new MCP server to replace the default one
+   */
+  mcpServerSetup?: (server: McpServer, farmServer: Compiler) => Awaitable<void | McpServer>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)